Dockerfile.ocp

```diff
@@ -1,15 +1,14 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.11 AS builder
```

Comment:
Should be `FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder` probably?

Reply:
This will need to be verified by someone from ART. Both images (`registry.svc.ci.openshift.org/ocp/builder:golang-1.11` and `registry.svc.ci.openshift.org/openshift/release:golang-1.11`) are available in registry.

Reply:
For OCP, `FROM` directives are rewritten so I'm quite sure we need to keep `openshift` for OKD.

Reply:
Just tested, those images are the same (matching sha256 sums) just with different name:
```
$ docker pull registry.svc.ci.openshift.org/openshift/release:golang-1.11
golang-1.11: Pulling from openshift/release
Digest: sha256:2405aaa3ff648828f89c40e12a9a148957f7ae3c82312d057551da6d9820e8e1
Status: Downloaded newer image for registry.svc.ci.openshift.org/openshift/release:golang-1.11
$ docker pull registry.svc.ci.openshift.org/ocp/builder:golang-1.11
golang-1.11: Pulling from ocp/builder
Digest: sha256:2405aaa3ff648828f89c40e12a9a148957f7ae3c82312d057551da6d9820e8e1
Status: Image is up to date for registry.svc.ci.openshift.org/ocp/builder:golang-1.11
```

Reply:
Ack. For cleanness, it should be `openshift` then since `ocp` is the downstream.

Reply:
fixed :)